Tasks/AzureSpringCloudV0/deploymentProvider/azure-arm-spring-apps.ts

```diff
@@ -246,6 +246,10 @@ export class AzureSpringCloud {
                     throw ToError(response);
                 }
                 provisioningState = response.body.properties.provisioningState;
+                if (provisioningState == "Failed"){
+                    tl.debug('KPack build failed');
+                    throw ToError(response);
+                }
```

Comment:
[Suggestion] These changes aren't seems related to rebranding 😃
Probably, make sense to move it into separate PR, but it's up to you

Reply:
Thanks, sorry I didn't pointed this out, this would be a small ux improvement I think I could include here.